build/postinstall.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
What about Windows?

Reply:
Currently failing, but trying to see if this even works at all.

Reply:
It doesn't. Windows doesn't support bash.